tests/unit/config/test_config.py

```diff
@@ -356,6 +356,34 @@ def test_command_substitution_from_other_section(self, newconfig):
         x = reader.getargvlist("commands")
         assert x == [["echo", "whatever"]]
 
+    def test_skipsdist_factor_with_no_factor(self, newconfig):
+        config = newconfig(
+            """
+            [tox]
+            skipsdist = factor: true
+            [testenv]
+            commands =
+                echo {[tox]skipsdist}
+            """
+        )
+        reader = SectionReader("testenv", config._cfg)
+        x = reader.getargvlist("commands")
+        assert x == [["echo", "false"]]
+
+    def test_skipsdist_factor_with_factor(self, newconfig):
+        config = newconfig(
+            """
+            [tox]
+            skipsdist = factor: true
+            [testenv]
+            commands =
+                echo {[tox]skipsdist}
+            """
+        )
+        reader = SectionReader("testenv", config._cfg, factors={'factor'})
+        x = reader.getargvlist("commands")
+        assert x == [["echo", "true"]]
```

Comment:
This is essentially a single test with different params. So why not use parametrize?
```suggestion
    @pytest.mark.parametrize(
        ('section_reader_kwargs', 'expected_outcome'),
        (
            ({}, 'false'),
            ({'factors': {'factor'}}, 'true'),
        ),
        ids=('factor missing', 'factor present'),
    )
    def test_skipsdist_factor(
            self, newconfig,
            section_reader_kwargs, expected_outcome,
    ):
        config = newconfig(
            """
            [tox]
            skipsdist = factor: true
            [testenv]
            commands =
                echo {[tox]skipsdist}
            """
        )
        reader = SectionReader("testenv", config._cfg, **section_reader_kwargs)
        x = reader.getargvlist("commands")
        assert x == [["echo", expected_outcome]]
```